### ISSUE TITLE
Tests are failing

### ISSUE THREAD

**Issue by chrismytton** [mysociety/django-info-pages#6]
When I try and run the tests I get the following error:

```
$ python runtests.py
Traceback (most recent call last):
  File "runtests.py", line 51, in <module>
    runtests()
  File "runtests.py", line 44, in runtests
    django.setup()
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python@2/2.7.16/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/autocomplete_light/__init__.py", line 7, in <module>
    from .shortcuts import *  # noqa
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/autocomplete_light/shortcuts.py", line 8, in <module>
    from .forms import *
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/autocomplete_light/forms.py", line 438, in <module>
    class ModelForm(six.with_metaclass(*bases)):
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/autocomplete_light/forms.py", line 283, in __new__
    attrs)
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/forms/models.py", line 247, in __new__
    new_class = super(ModelFormMetaclass, mcs).__new__(mcs, name, bases, attrs)
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/forms/forms.py", line 91, in __new__
    .__new__(mcs, name, bases, attrs))
  File "/Users/chris/.local/share/virtualenvs/django-info-pages/lib/python2.7/site-packages/django/forms/widgets.py", line 145, in __new__
    .__new__(mcs, name, bases, attrs))
TypeError: Error when calling the metaclass bases
    metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
```

**Comment by chrismytton** [comment]
I had `six==1.12.0` installed, downgrading to `six==1.10.0` and then running `pip install -e '.[test]'` (to get the test dependencies) has got the tests passing again.

**Comment by chrismytton** [comment]
Looks like this is actually a problem in django-autocomplete-light, and is fixed in [version 2.3.6](https://github.com/yourlabs/django-autocomplete-light/blob/74be7a95b446700804f911aaa87ab013f866ba60/CHANGELOG#L1).

(Strangely that version isn't actually listed in the [`CHANGELOG` on the master branch](https://github.com/yourlabs/django-autocomplete-light/blob/ab632e4d74f73e0f800f2c403e2f19436a498943/CHANGELOG#L236) 😕)

**Comment by chrismytton** [comment]
Ah, except newer versions of django-autocomplete-light only work with Django 1.11 (https://github.com/yourlabs/django-autocomplete-light/commit/2297a8083bdcb45bd051b0042082237ee0795c9e), which we're not on yet. So I think the best solution for now is to explicitly pin the version of six that we want.

**Comment by chrismytton** [comment]
Fixed in #7
